libcxx/include/__config

```diff
@@ -174,11 +174,6 @@
 // The implementation moved to the header, but we still export the symbols from
```

Comment:
Not attached to this line: the commit message should be self explanatory -- it shouldn't only contain a reference to the original discussion.

Reply:
You mean the first comment on github or were you saying the actual message attached to the first commit wasn't sufficient?

I changed the first github comment to match the commit message with a little more info.

Reply:
I meant the PR description on Github, since that is what Github uses to generate the commit message when we "squash and merge". Surprisingly, it seems that the actual commit messages are completely disregarded for that purpose.

Reply:
Resolving. Please re-open if i misunderstood.

Reply:
Urg, well thats gross and bad design on the part of github. 

I'll remember that going forward.